src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx

```diff
@@ -296,19 +296,23 @@ const BlastSettings = ({ config }: Props) => {
   );
 };
 
-const BlastJobName = () => {
-  const jobName = useAppSelector(getBlastJobName);
-  const { setBlastJobName } = useBlastForm();
+const BlastSubmissionName = () => {
+  const submissionName = useAppSelector(getBlastSubmissionName);
+  const { setBlastSubmissionName } = useBlastForm();
 
   const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const name = e.currentTarget.value;
-    setBlastJobName(name);
+    setBlastSubmissionName(name);
   };
 
   return (
-    <div className={styles.blastJobName}>
-      <span>Name this job</span>
-      <ShadedInput value={jobName} onChange={onChange} placeholder="optional" />
+    <div className={styles.blastSubmissionName}>
+      <span>Submission name</span>
```

Comment:
Oh, interesting! I didn't know Andrea has changed this label text; but now I can see in an [XD](https://xd.adobe.com/view/879a4244-8420-48c1-b3c9-c998e77651fd-d2c9/screen/571dfdb2-6465-4a1d-9d0f-300cf632aa27?fullscreen) that she has. Hoping I am looking at the up-to-date XD 😄 

However, you will notice that the style of this label in the XD is different; and that the placeholder text in the input starts with a capital "O".

Reply:
Done.